prow/plugins/blunderbuss/blunderbuss.go

```diff
@@ -251,6 +257,7 @@ func handle(ghc githubClient, roc repoownersClient, log *logrus.Entry, reviewerC
 func getReviewers(rc reviewersClient, ghc githubClient, log *logrus.Entry, author string, files []github.PullRequestChange, minReviewers int, useStatusAvailability bool) ([]string, []string, error) {
 	authorSet := sets.NewString(github.NormLogin(author))
 	reviewers := sets.NewString()
+	reviewersList := []string{}
```

Comment:
`var reviewersList []string` (reviewersList = append(reviewersList, "foo") works just fine)

On the other hand I don't think we need any of the changes in `getReviewers()` -- it is just a clone of reviewers